webapp/cypress/integration/register.feature

```diff
@@ -0,0 +1,98 @@
+
+  Scenario: Alert is displayed when I enter not valid full name
+    Given I enter "test@test.pl" into "Email" field on the "Register" Page
+    And I enter "NaMe@123&" into "Full name" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Password" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Repeat your password" field on the "Register" Page
+    When I click on "Register" button on the "Register" Page
+    Then Alert "Full name not valid!" is displayed under "Full name" field on the "Register" Page
+
+  Scenario: Alerts is displayed when I left password form empty
+    Given I enter "test@test.pl" into "Email" field on the "Register" Page
+    And I enter "Test User" into "Full name" field on the "Register" Page
+    And I left empty "Password" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Repeat your password" field on the "Register" Page
+    When I click on "Register" button on the "Register" Page
+    Then Alert "Password is required!" should be displayed under "Password" field on the "Register" Page
+    And Alert "Passwords does not match!" is displayed under "Repeat your password" field on the "Register" Page
+
+  Scenario: Alerts is displayed when I left password and repeat your password forms empty
+    Given I enter "test@test.pl" into "Email" field on the "Register" Page
+    And I enter "Test User" into "Full name" field on the "Register" Page
+    And I left empty "Password" field on the "Register" Page
+    And I left empty "Repeat your password" field on the "Register" Page
+    When I click on "Register" button on the "Register" Page
+    Then Alert "Password is required!" is displayed under "Password" field on the "Register" Page
+    And Alert "Passwords does not match!" is displayed under "Repeat your password" field on the "Register" Page
+
+  Scenario: Alert is displayed when I left repeat your password form empty
+    Given I enter "test@test.pl" into "Email" field on the "Register" Page
+    And I enter "Test User" into "Full name" field on the "Register" Page
+    And I enter "Pa$$word123@" into "Password" field on the "Register" Page
+    And I left empty "Repeat your password" field on the "Register" Page
+    When I click on "Register" button on the "Register" Page
+    Then Alert "Passwords does not match!" is displayed under "Repeat your password" field on the "Register" Page
+
+  Scenario: Alert is displayed when I enter not valid password
+    Given I enter "test@test.pl" into "Email" field on the "Register" Page
+    And I enter "Test User" into "Full name" field on the "Register" Page
+    And I enter "password" into "Password" field on the "Register" Page
+    And I enter "password" into "Repeat your password" field on the "Register" Page
+    When I click on "Register" button on the "Register" Page
+    Then Alert:
+        """
+        "Password must contain 8 characters, one uppercase, one lowercase, one number and one special case character!"
```

Comment:
pamiętaj żeby sprawdzić przy implementacji czy to na pewno działa

Reply:
test failował, wiec zmieniłem